packages/vscode-extension/src/chat/prompts.ts

```diff
@@ -11,7 +11,7 @@ export const defaultSystemPrompt = () => {
   );
 
   return new vscode.LanguageModelChatMessage(
-    vscode.LanguageModelChatMessageRole.System,
+    vscode.LanguageModelChatMessageRole.User,
```

Comment:
I think we could also define a new alias type so that we are able to switch back to system role easily in the future.

Reply:
It is unnecessary because the proposed api may not be stable in the future.